module/VuFind/tests/integration-tests/src/VuFindTest/Mink/SearchFacetsTest.php

```diff
@@ -1146,4 +1244,110 @@ public function testOrFacets(): void
             $this->findCssAndGetText($page, '.search-header .search-stats')
         );
     }
+
+    /**
+     * Data provider for testCheckboxFacets
+     *
+     * @return array
+     */
+    public static function checkboxFacetselectionProvider(): array
```

Comment:
Inconsistent capitalization:

```suggestion
    public static function checkboxFacetSelectionProvider(): array
```

Reply:
Fixed.